chapters/refereecommands.adoc

```diff
@@ -17,8 +17,25 @@ When the halt command is issued, no robot is allowed to move or <<Ball Manipulat
 There is a grace period of 2 seconds for the robots to brake.
 
 .Usage
-The halt command allows the referee to interrupt the game immediately whenever an emergency occurs (for example when a robot gets out of contol). It is
-also used to recalibrate the vision software during a game if the vision expert considers it necessary and the referee agrees and for <<Robot Substitution, robot substitution>>. Additionally, the referee is free to issue the halt command at will.
+The halt command allows the referee to interrupt the game immediately whenever an emergency occurs (for example when a robot gets out of control). It is also used for <<Robot Substitution, robot substitution>>. Additionally, the referee is free to issue the halt command at will.
+
+Regarding the vision system, the game will be immediately halted in case of catastrophic failures, which includes:
+
+. Camera failures;
+. SSL-Vision crash;
+. Network failures;
+. Anything that causes a significant drop in the frame rate of the data;
+
+The game is continued in case of:
+
+. The ball is detected at least every 0.2 seconds while not being covered by some robot or shadow;
+. All robots on the field are detected at least every 0.5 seconds;
+
+In case the robot or ball detection gets worse than the previously defined threshold, the game may be halted. There won't be any tool that guarantees that the game will be halted in case those thresholds are violated.
```

Comment:
There will probably be a tool (https://github.com/RoboCup-SSL/ssl-quality-inspector), so the formulation is not ideal.

Maybe something like: The vision expert may monitor a quality inspecting tool (such as https://github.com/RoboCup-SSL/ssl-quality-inspector) that both teams agree with and signal issues with the vision system to the referee. Only tools that are run on official league computers are allowed for vision quality checks.

Maybe also add a note box explaining the reasoning behind this (rough formulation): Games should not be interrupted just because teams think that there may be an issue. Only officially proven issues should lead to further investigation.